resource/airflow_dag_pipeline.yaml

```diff
@@ -49,6 +49,207 @@ jobs:
       AWS_NODES_ROLE_ARN: ((secrets.nodes-role-arn))
       APP_NAME: ((app-name))
 
+- name: test
+  plan:
+  - get: pull-request
+    trigger: true
+    version: every
+
+  - get: common-tasks
+
+  # Update github 'checks' on PR
+  - aggregate:
+    - put: pull-request
+      params:
+        path: pull-request
```

Comment:
these status checks sometimes fail can you add attempts: 3 to anything that could possibly fail but shouldn't kill the build? https://concourse-ci.org/attempts-step-modifier.html

Reply:
Added attempts to all the github status updates